test.js

```diff
@@ -1,5 +1,5 @@
 import test from 'tape';
-import insertText from '.';
+import {insert, set, getSelection, wrapSelection} from '.';
```

Comment:
I don't really like how these are named imports. They make no sense without looking at the import. "set" what? I would personally just do a default export and attach all of them to the default export. 🤷‍♂️

Reply:
Me neither, but XO was complaining at some point, I think https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-named-as-default.md

This is because at default I also have an object that includes all imports (so one doesn’t have to use `* as textFieldEdit`)

I guess (short of disabling the rule) I could use:

``` ts
export function insertTextInField(){};
export function setFieldValue(){};
export function wrapSelectionInField(){};
export function getSelectionFromField(){};

export default {
  insert: insertText,
  … etc
}
```

Reply:
```js
import textFieldEdit from '.';
```

causes

```
  ✖   19:2   Caution: textFieldEdit also has a named export insert. Check if you meant to write import {insert} from '.' instead.                import/no-named-as-default-member

**** repeated for all **
```

I can't just disable the rule because dependants will encounter it too.

I think an easy solution would be to drop the `default` export and have it used as:

```js
import * as textFieldEdit from '.';
```

Like I did in `indent-text-area`: https://github.com/sindresorhus/refined-github/blob/d68213c088ff3d3f573a77f3cfd5f087756023e5/source/options.tsx#L8

Related: https://github.com/sindresorhus/meta/issues/6